### DISCUSSION TITLE
怎么加一个ignore/DIRECT的网址？

### DISCUSSION THREAD

**Discussion by Lihael** [MetaCubeX/ClashMetaForAndroid#381] [Q&A]
不知道因为什么原因，我只要是开了clash 我就上不了 torrentio.strem.fun. 我想把这个网页加入个黑名单，让clash 直接ignore掉，有什么办法？

我之前试了在规则yaml里面加了个 rules: - DOMAIN-SUFFIX,strem.fun,DIRECT 可是会报错 "mapping rules are not allowed in this context". 有其他方法吗？

**Comment by xiaofeng0223** [comment]
解决了吗
